src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -1061,6 +1126,26 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         }
     });
 
+    readonly mutationProfileDuplicateSamplesReport = remoteData({
+        await:()=>[
+            this.horzAxisDataPromise,
+            this.vertAxisDataPromise
+        ],
+        invoke:()=>{
+            return Promise.resolve(getMutationProfileDuplicateSamplesReport(
+                this.horzAxisDataPromise.result!,
+                this.vertAxisDataPromise.result!,
+                this.horzSelection,
+                this.vertSelection
+            ));
+        },
+        default:{
```

Comment:
why? can't we just check status?

Reply:
the defaultValue was really put in place in mobxpromise to solve the annoying problem of having to put ! after result reference (even when we check complete status)

Reply:
sure